### ISSUE TITLE
Rule proposal: `prefer-await-ops`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1456]
[await.ops proposal
](https://github.com/tc39/proposal-await.ops)

## Fail

```js
await Promise.all(users.map(async x => fetchProfile(x.id)))
```

## Pass

```js
await.all users.map(async x => fetchProfile(x.id))
```

The proposal is still stage 1, we need wait at least we can test on some parser.
